optimism/src/keccak/environment.rs

```diff
@@ -17,7 +17,7 @@ pub struct KeccakEnv<Fp> {
     /// Constraints that are added to the circuit
     pub(crate) constraints: Vec<E<Fp>>,
     /// Values that are looked up in the circuit
-    pub(crate) _lookup_terms: [Vec<Lookup<E<Fp>>>; 2], // at most 2 values are looked up at a time
+    pub(crate) _lookup_terms: Vec<Lookup<E<Fp>>>, // at most 5 values are looked up at a time
```

Comment:
I would prefer to keep a fixed-length vector, and simply increase the size.

Reply:
The old type of an array was a typo